gatsby-config.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
         short_name: `ethereum.org`,
         start_url: `/en/`,
         background_color: `#ffffff`,
-        theme_color: `#1c1ce1`,
+        theme_color: `#222222`,
```

Comment:
can be shortened to `#222`

Reply:
you could make that argument for the variable above too, I think it's fairly non-standard even if you can do this in the context of hex codes.

Leaving as a 6-digit colour code allows developers to more easily grok that this is a color.

Reply:
Yeah, we recently tried to standardize the codebase to always use `#rrggbb` syntax instead of shortening them.

Reply:
I think we settled on the 6 character standard for the project for consistency.